docs/user_documentation.md

```diff
@@ -438,34 +438,55 @@ You can configure the action by specifying the ArtifactID of the destination fol
 
 ![](media/4ea5493c6871466e5ab385ec5c08f8ff.png)
 
-### Data Archive Action Type
+### Data Disposal Action Type
 
-> **WARNING:** The Data Archive Action will permanently delete all documents that match the Rule conditions and are outside the Data Retention window.
+> **WARNING:** The Data Disposal Action will permanently delete all documents that match the Rule conditions and are outside the Data Retention window.
+>
+> Data Disposal will only delete files located in valid Data Batch folders. Documents not ingested by Trace will not be deleted
+>
+> If a Document is disposed, but not its parent or children, only the disposed Documents files are deleted
+>
+> Documents which were imported as part of a Data Batch which is in the `CompletedWithErrors` state will not be deleted
 
-The Data Archive Action Type follows the same Trace Rules Engine paradigm with
+The Data Disposal Action Type follows the same Trace Rules Engine paradigm with
 one added condition:
 
--   You attach the Data Archive action to a Rule
+-   You attach the Data Disposal action to a Rule
 
 -   The rule executes the actions on documents that match the Rule conditions
 
     -   Searchable Set
 
     -   Term Searching (optional)
 
-    -   *SPECIFIC TO ARCHIVE*: `Delete Documents Older Than Hours` setting
+    -   *SPECIFIC TO DISPOSAL*: `Delete Documents Older Than Hours` setting
 
 Any document that is newer than the specified number of hours (based on System
 Created On date/time) will not be deleted even if the document is included in the
-Searchable Set / Search Term
+Searchable Set / Search Term.
+
+This action is used to:
+1. Clear out old documents from the Trace workspace
+2. Free disk space by deleting **ALL** disposed Document files from the File Share
 
 **Action Configuration**
 
 `Document Delete Batch Size` - controls number of documents to delete at once
+> **Note:** All documents will be deleted in a single pass, this is a tweak to improve SQL performance
+
+`Files On Disk Document Delete Page Size` - controls the number of Relativity Documents to process file deletion for at a time
 
 `Delete Documents Older Than Hours` - controls age of a document (based on System
 Created On date/time) to delete
 
+**Saved Search Recommendations for Data Disposal**
+
+Because of the risk of data loss. You should carefully configure the Searchable Set used for Data Disposal. The following are recommended minimum filtering parameters
+
+-   Trace Has Errors is False
+-   A field marking the document as Responsive is False
```

Comment:
Not sure what the industry uses as a term for Responsive, but we should use that term here